src/Lykke.Service.Assets/Modules/CqrsModule.cs

```diff
@@ -64,44 +59,39 @@ protected override void Load(ContainerBuilder builder)
 
                 return new CqrsEngine(ctx.Resolve<ILogFactory>(),
                     ctx.Resolve<IDependencyResolver>(),
-#if DEBUG
-                    new MessagingEngine(ctx.Resolve<ILogFactory>(),
-                        new TransportResolver(new Dictionary<string, TransportInfo>
-                        {
-                            {"RabbitMq", new TransportInfo(rabbitMqSettings.Endpoint + virtualHost, rabbitMqSettings.UserName, rabbitMqSettings.Password, "None", "RabbitMq")}
-                        }),
-                        new RabbitMqTransportFactory(ctx.Resolve<ILogFactory>())),
-#else
                     new MessagingEngine(ctx.Resolve<ILogFactory>(),
```

Comment:
It is important to dispose MessagingEngine but in the current implementation is will never be disposed. Register it in the container.